piggybank/src/com/company/Money.java

```diff
@@ -0,0 +1,31 @@
+package com.company;
+
+public abstract class Money {
+    private double value;
+    private String name;
+    private int quantity;
+
+    public void setValue(double value ){
```

Comment:
Interesting that you use a setter for value. I'm not sure if that would make the most sense, since the value should be dependent on the value of your coin and the quantity that you have. It may make the most sense to only have a `getValue()` that returns value * quantity. That way, you only need to change the quantity and your value would still remain accurate